### PR TITLE
first draft pages and updates added

### PR DESCRIPTION
**Pull request by ntner**
Added sub pages for rack parameters and cloud providers
Added pages for all v3 provider rack params
Updated CLI page with new cloud provider params

### CODE REVIEW

File: docs/configuration/rack-parameters/aws/availability_zones.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- **Controlled Resource Distribution**: By specifying AZs, you can control how your resources are distributed across the AWS region, which can be important for compliance or operational reasons.
+- **High Availability**: Ensuring that your applications and services are spread across multiple AZs enhances fault tolerance and availability.
+
+## Setting Parameters
+To set the `availability_zones` parameter, use the following command:
+```html
+$ convox rack params set availability_zones=us-east-1a,us-east-1b,us-east-1c -r rackName
+Setting parameters... OK
+```
+This command sets the availability zones to `us-east-1a`, `us-east-1b`, and `us-east-1c`.
```
Comment (nightfury1204) [inline docs/configuration/rack-parameters/aws/availability_zones.md:26]: Not sure it is safe to set after installing the rack

File: docs/configuration/rack-parameters/aws/private.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- **Compliance Requirements**: Certain compliance standards may require that sensitive workloads run in private subnets.
+
+## Setting Parameters
+To set the `private` parameter, use the following command:
+```html
+$ convox rack params set private=false -r rackName
+Setting parameters... OK
+```
+This command sets the nodes to be placed in public subnets.
+
```
Comment (nightfury1204) [inline docs/configuration/rack-parameters/aws/private.md:29]: at the install time

File: docs/configuration/rack-parameters/aws/vpc_id.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- **Existing VPC Integration**: Use this parameter to integrate your Convox rack with an existing VPC.
+- **Custom Network Configuration**: Specify an existing VPC to meet specific network requirements and configurations.
+
+## Setting Parameters
+To set the `vpc_id` parameter, use the following command:
+```html
+$ convox rack params set vpc_id=vpc-12345678 -r rackName
+Setting parameters... OK
+```
+This command sets the VPC ID to the specified value.
```
Comment (nightfury1204) [inline docs/configuration/rack-parameters/aws/vpc_id.md:26]: At the install time only

Comment (ntner) [reply docs/configuration/rack-parameters/aws/vpc_id.md]: fixed

File: docs/configuration/rack-parameters/aws/private.md (hunk 2 above)
Comment (ntner) [reply docs/configuration/rack-parameters/aws/private.md]: fixed

File: docs/configuration/rack-parameters/aws/cidr.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## Setting Parameters
+To set the `cidr` parameter, use the following command:
+```html
+$ convox rack params set cidr=10.2.0.0/16 -r rackName
+Setting parameters... OK
+```
+This command sets the CIDR range for the VPC to `10.2.0.0/16`.
+
+## Additional Information
```
[the remark this answers is not in the document]
Comment (ntner) [reply docs/configuration/rack-parameters/aws/cidr.md]: fixed

File: docs/configuration/rack-parameters/aws/internet_gateway_id.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- **Existing VPC Integration**: Use this parameter to integrate your Convox rack with an existing VPC that has an internet gateway attached.
+- **Network Configuration**: Ensures that your rack can access the internet through the specified internet gateway.
+
+## Setting Parameters
+To set the `internet_gateway_id` parameter, use the following command:
+```html
+$ convox rack params set internet_gateway_id=igw-1234567890abcdef0 -r rackName
+Setting parameters... OK
+```
+This command specifies the ID of the internet gateway for your existing VPC.
```
[the remark this answers is not in the document]
Comment (ntner) [reply docs/configuration/rack-parameters/aws/internet_gateway_id.md]: fixed

File: docs/configuration/rack-parameters/aws/availability_zones.md (hunk 1 above)
Comment (ntner) [reply docs/configuration/rack-parameters/aws/availability_zones.md]: fixed

**Pull request merged**